typescript/sdk/src/index.ts

```diff
@@ -251,4 +250,13 @@ export {
 export { MultiGeneric } from './utils/MultiGeneric';
 export { filterByChains } from './utils/filter';
 export { multisigIsmVerificationCost } from './utils/ism';
+export {
+  AccountDataWrapper,
+  HypTokenInstruction,
+  HyperlaneTokenData,
+  HyperlaneTokenDataSchema,
+  TransferRemoteInstruction,
+  TransferRemoteSchema,
+  TransferRemoteWrapper,
+} from './utils/sealevel/tokenSerialization';
```

Comment:
I would be inclined to keep this serialization code adjacent to the adapter file, since that'll likely be it's only consumer.

Reply:
it's used by the router adapter and the token adpater